### DISCUSSION TITLE
QueryFilter Attribute example?

### DISCUSSION THREAD

**Discussion by mxmissile** [linq2db/linq2db#4541] [Q&A] [answered]
Is there an example of using this attribute and with setting the FilterLambda property?

https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/Mapping/QueryFilterAttribute.cs/

` [QueryFilter(FilterLambda=...)]` is not valid syntax

**Comment by MaceWindu** [comment, marked as answer]
Looks like bug https://github.com/linq2db/linq2db/issues/4543
you can use fluent mapping for now to specify it
